libraries/node-core-library/src/FileError.ts

```diff
@@ -43,10 +48,10 @@ export class FileError extends Error {
   }
 
   /** @override */
-  public toString(format: FileErrorFormat = FileErrorFormat.Unix): string {
+  public toString(format: FileErrorFormat = 'Unix'): string {
```

Comment:
The formatter should really be a function called on the error, not built into the error implementation. That way it is extensible.
As a basic example, it is useful to format file errors when on AzureDevOps by emitting the `##vso[task.logissue type=error;sourcepath=${error.filePath};linenumber=${error.line};columnnumber=${error.column};]${error.message}` notation

Reply:
Spoke offline. Leaving as-is for now, since this PR is largely about moving existing code. I will however add support for Azure DevOps formatting, and defaulting to said formatting on Azure DevOps pipelines unless another format is specified.